messaging/ses/src/sendEmail.graphql

```diff
@@ -0,0 +1,14 @@
+type SesEmailPayload {
+  success: Boolean!
+}
+
+extend type Mutation {
+  sendSesEmail(
+    from: String!
+    to: [String!]!
+    subject: String!
+    text: String!
+    html: String!
```

Comment:
are both `text` and `html` always required? or do you have to choose one of them?

Reply:
No, just one is required. Either you can choose `html` or `text` or both. If we take one away, I'd vote on `text`

Reply:
In this case, please mark both strings as optional:

```graphql
text: String
html: String
```

and check in code. reject if both are specified

Reply:
Why reject if both are specified? `text` is a fallback if `html` fails